### ISSUE TITLE
dont save face

### ISSUE THREAD

**Issue by Nerauh76** [xAranaktu/FIFA-23-Live-Editor#302]
Hello, 
when I modify my hair and add tattoos in the f9 it works but as soon as I start a match or a training it goes back to the original, is this normal?

**Comment by theoriginalskengman** [comment]
Are you saving the work you've done on LE before going back into the game??

**Comment by Nerauh76** [comment]
Yes , i have save and after playing match or training its not save > LE

**Comment by Teriphery** [comment]
Same issues here. Even if you change any attribute/appearance
